app/gzac/src/main/java/com/ritense/gzac/listener/ApplicationReadyEventListener.kt

```diff
@@ -22,6 +25,20 @@ class ApplicationReadyEventListener(
     fun handle() {
         connectorService.getConnectorTypes().forEach {
             try {
+                if (it.name == "OpenZaak") {
+                    connectorService.createConnectorInstance(
+                        typeId = it.id.id,
+                        name = it.name.replace("\\s".toRegex(), "") + "Instance",
```

Comment:
Als je een mogelijke spatie in de naam verwacht dan klopt de if-statement op regel 28 ook niet toch?

Reply:
Aangepast